01_deploy_nodes.py

```diff
@@ -133,6 +134,29 @@ def run(self):
             pass
         os.symlink(self.result_dir, link)
 
+        ## create ssh tunnels file
+        logger.info("ssh tunnel informations:")
+        logger.info("___")
+        
+        script = "cat > /tmp/openstack_ssh_config <<EOF\n"
```

Comment:
Actually the file should be copied on your local machine (i.e. your laptop/workstation) so I'm not sure the comment is relevant here.